src/locales/us/en/index.json

```diff
@@ -44,7 +45,6 @@
   "securityQuestions": "Security question(s)",
   "securityQuestionsPrompt": "Provide custom security question(s) and answer(s):",
   "showPassword": "Show password",
-  "signInWith": "Sign in with",
```

Comment:
I replaced the default button text "Sign in with", with "Continue with" since it seems to be more applicable in scenarios other than logging in, e.g. registration. However further to my comment a bit further up about branding of buttons for different IDPs, we may need to show "Sign in with Apple", "Continue with Facebook", etc. This may get a bit messy. Not sure what we can do about this but it feels like we should be consistent in our defaults ("Continue with.." would be my preference) - and allow developers to override in the localization files.

Reply:
Tough question. I think "Continue with" is the best choice. I looked through the guidelines of all three and I don't think they are super rigid on the text preceding the company name. This make sense as it would have to accommodate localization. Not every language is going to literally translate to "Sign in with" or whatever. I think they just want it to be descriptive of the action that is triggered by the button.